src/views/AccountDetails/AccountDetails.js

```diff
@@ -568,28 +570,10 @@ class AccountDetails extends React.Component {
                 value={latestPaymentStatus}
               />
             </Col>
-            <Col
-              data-test-overdue-policy
-              xs={1.5}
-            >
-              <KeyValue
-                label={<FormattedMessage id="ui-users.loans.details.overduePolicy" />}
-                value={overdueFinePolicyId
-                  ? <Link to={`/settings/circulation/fine-policies/${overdueFinePolicyId}`}>{overdueFinePolicyName}</Link>
-                  : '-'
-                }
-              />
-            </Col>
-            <Col
-              data-test-lost-item-policy
-              xs={1.5}
-            >
+            <Col xs={1.5} sm={3} md={3} lg={3}>
               <KeyValue
-                label={<FormattedMessage id="ui-users.loans.details.lostItemPolicy" />}
-                value={lostItemPolicyId
-                  ? <Link to={`/settings/circulation/lost-item-fee-policy/${lostItemPolicyId}`}>{lostItemPolicyName}</Link>
-                  : '-'
-                }
+                label={<FormattedMessage id="ui-users.feefines.details.dateClose" />}
+                value="-"
```

Comment:
Please use [`<NoValue>`](https://github.com/folio-org/stripes-components/tree/master/lib/NoValue) instead of `-`; it's better for accessibility.

Reply:
The screenshots attached to the story imply the "Date fee/fine closed" field should display a timestamp but that isn't the case here. Shouldn't `value` be a ternary, like the others, e.g. something like
```
value={loan.closedDate ? <FormattedTime value={loan.closedDate} /> : <NoValue />}
```

Reply:
Apply the recommendations or fixes in my new commit